contracts/defter.sol

```diff
@@ -177,5 +184,7 @@ contract Defter {
         }
     }
 
+    receive() external payable {}
```

Comment:
receive and fallback functions are nod required

Reply:
done